extensions/logs/logs.go

```diff
@@ -125,6 +126,20 @@ func ClusterDump(suiteName, testName string) {
 			if err != nil {
 				logrus.Errorf("An error while getting cluster dump. Error: %s", err.Error())
 			}
+
+			allContainers, _, _, _ := runner.Run(`docker ps --format '{{.Names}}'`)
+			containers := filterContainers(allContainers)
+
+			for _, container := range containers {
+				_, _, dExitCode, dErr := runner.Run(fmt.Sprintf("docker logs %s > %s/%s.log", container, suitedir, container))
```

Comment:
artifacts dir is missing. Please have a look as it implemented for cluster dump above

Reply:
artifacts dir is included in the suitedir as it done above, I use suitedir